src/ContainerDetails.jsx

```diff
@@ -12,6 +12,14 @@ const render_container_state = (container) => {
     return cockpit.format(_("Exited"));
 };
 
+const render_container_published_ports = (container) => {
+    if (container.ports)
+        return container.ports
+                .map(port => port.host_ip ? port.host_ip : '0.0.0.0' + ':' + port.host_port + '->' + port.container_port + '/' + port.protocol)
```

Comment:
Please use `a || b` instead of `a ? a : b`.

Reply:
I'll just remove the map maybe.